pkgs/top-level/aliases.nix

```diff
@@ -1548,7 +1548,8 @@ mapAliases ({
   youtubeDL = throw "'youtubeDL' has been renamed to/replaced by 'youtube-dl'"; # Converted to throw 2022-02-22
   ytop = throw "ytop has been abandoned by upstream. Consider switching to bottom instead";
   yubikey-neo-manager = throw "yubikey-neo-manager has been removed because it was broken. Use yubikey-manager-qt instead"; # Added 2021-03-08
-  yuzu = yuzu-mainline; # Added 2021-01-25
+  yuzu-ea = yuzu-early-access; # Added 2022-08-18
+  yuzu = throw "'yuzu' has been renamed to 'yuzu-mainline'"; # Converted to throw throw 2022-08-18
```

Comment:
I don't see the point of throwing, it's unfriendly when we have an obvious thing to remap it to.

Reply:
I was thinking that its just of a pointless alias, added around a year ago purely because another branch was introduced. It was added back then for backwards compatibility, and seeing how a while has passed now I figured I'd just remove it.

I don't feel strongly about this though, I'll add the alias back shortly 👍

I'm also curious; I've have seen a fair bit of other aliases being converted to a throw as well, would you consider all of those unfriendly?

Reply:
Dropped the commit :+1:

Reply:
> I was thinking that its just of a pointless alias, added around a year ago purely because another branch was introduced. It was added back then for backwards compatibility, and seeing how a while has passed now I figured I'd just remove it.

I only recently started using Yuzu, and I think being able to `nix-shell -p yuzu` without being aware of the different branches is worth the alias.